### PR TITLE
Restyle according to new brand guidelines

### PR DESCRIPTION
**Pull request by ajcoppa**
Screenshot 2023-04-01 at 12 20 10 PM

### CODE REVIEW

**Review by JamesDowney — COMMENTED** [review]
This is the change we needed.

File: Source/relay/TourGuide/Main.ash [hunk 1]
```diff
@@ -62,6 +61,9 @@ void runMain(string relay_filename)
 	generateChecklists(ordered_output_checklists);
 	
     string guide_title = "TourGuide";
+    if (isAprilFools()) {
+        guide_title = "Glup Shitto Stole Rogue V's Targeting Computer";
```
Comment (docrostov) [inline Source/relay/TourGuide/Main.ash:65]: this is great

File: Source/relay/TourGuide/Main.ash [hunk 2]
```diff
@@ -262,7 +268,12 @@ void runMain(string relay_filename)
         // Head text
         
         // Title
-        PageWrite(HTMLGenerateSpanOfStyle(guide_title, "font-weight:bold; font-size:1.5em"));
+        string titleStyles = "font-weight:bold; font-size:1.5em;";
+        if (isAprilFools()) {
+            titleStyles = titleStyles + "background-image: linear-gradient(-225deg,#231557 0%,#44107a 15%,#ff1361 30%,#fff800 60%);background-size: auto auto;background-clip: border-box;background-size: 200% auto;color: #fff;background-clip: text;text-fill-color: transparent;-webkit-background-clip: text;-webkit-text-fill-color: transparent;display: inline-block";
```
Comment (docrostov) [inline Source/relay/TourGuide/Main.ash:273]: you went unbelievably HAM on this and i appreciate it

**Review by docrostov — APPROVED** [review]
approved

**Pull request merged**